plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/driver/DriverDescriptor.java

```diff
@@ -1274,8 +1275,14 @@ public void loadDriver(DBRProgressMonitor monitor, boolean forceReload)
                 try {
                     // Load driver classes into core module using plugin class loader
                     driverClass = Class.forName(driverClassName, true, classLoader);
-                } catch (Throwable ex) {
-                    throw new DBException("Error creating driver '" + getFullName() + "' instance.\nMost likely required jar files are missing.\nYou should configure jars in driver settings.\n\nReason: can't load driver class '" + driverClassName + "'", ex);
+                } catch (Throwable e) {
+                    throw new DBCDriverException("Driver libraries (.jars) is not supplied in the product delivery"
+                        + " or libraries (.jars) not found in path."
+                        + "\nDriver configuration is not valid",
```

Comment:
What's "product delivery"? This doesn't sound like a sentence written in English.

Reply:
Mostly used term 
https://learningloop.io/glossary/product-delivery#:~:text=Product%20Delivery%20is%20a%20term,and%20quality%20of%20software%20development.

Reply:
I don't see this text on the screenshot below. Is it used somewhere else?

Reply:
Discussed internally to keep original text